tests/infrastructure/instance_types/test_cpu_memory_hotplug_instancetype.py

```
@@ -132,8 +132,10 @@ def test_hotplug_cpu_instance_type(self, instance_type_hotplug_vm, hotplugged_si
 
     @pytest.mark.dependency(depends=[f"{TESTS_CLASS_NAME}::hotplug_cpu_instance_type"])
     @pytest.mark.polarion("CNV-11402")
-    def test_migrate_snapshot_hotplugged_vm(self, hotplug_vm_snapshot_instance_type, instance_type_hotplug_vm):
-        migrate_vm_and_verify(vm=instance_type_hotplug_vm, check_ssh_connectivity=True)
+    def test_migrate_snapshot_hotplugged_vm(
+        self, admin_client, hotplug_vm_snapshot_instance_type, instance_type_hotplug_vm
```

Ruff (0.15.14)

[warning] Unused method argument: `hotplug_vm_snapshot_instance_type`

(ARG002)